code/game/mecha/equipment/tools/tools.dm

```diff
@@ -226,6 +226,70 @@
 	//ORIENTED
 	//PROGRAMMING
 
+/obj/item/mecha_parts/mecha_equipment/tool/scythe
+	name = "Heavy Duty Pneumatic Scythe"
+	desc = "An extremely heavy-duty pneumatic scythe. The \"giant robot\" approach to weed control. (Can be attached to: Engineering Exosuits)"
+	icon_state = "mecha_lazyscythecopypaste"
+	equip_cooldown = 20
+	energy_drain = 15
+	var/dam_force = 20
+
+/obj/item/mecha_parts/mecha_equipment/tool/scythe/can_attach(obj/mecha/working/ripley/M as obj)
+	if(..())
+		if(istype(M))
+			return 1
+	return 0
+
+/obj/item/mecha_parts/mecha_equipment/tool/scythe/action(atom/target)
+	if(!action_checks(target)) return
+
+	if(istype(target, /obj/machinery/portable_atmospherics/hydroponics))
+		set_ready_state(0)
+		if(do_after_cooldown(target, 1/2))
+			chassis.visible_message("<font color='red'><b>[chassis] smashes apart \the [target]!</b></font>", "You hear a pneumatic screeching and something being smashed apart.")
+			occupant_message("<font color='red'><b>You smash apart \the [target]!</b></font>")
+			log_message("Destroyed [target].")
+			var/obj/machinery/portable_atmospherics/hydroponics/tray = target
+			tray.smashDestroy(50) //Just to really drive it home
+			playsound(target, 'sound/mecha/mechsmash.ogg', 50, 1)
+	else if(istype(target, /obj/effect/plantsegment) || istype(target, /obj/effect/alien/weeds) || istype(target, /obj/effect/biomass)|| istype(target, /turf/simulated/floor))
+		set_ready_state(0)
+		var/olddir = chassis.dir
+		var/eradicated = 0
+		spawn for(var/i=1, i<=4, i++)

```

Comment:
N5X or Riri or some other nerd said that we should prolly just use 3 argument loops since they teach people how to make loops without magic syntax and aren't that ugly anyways. Considering that spess is literally the first coding experience for many people trying to make PRs I am not against that idea